src/components/DagreGraph/useCoordinatesCalculator.ts

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useState } from 'react';
+import { graphlib, Node } from 'dagre';
+import { Edges } from 'graph-model';
+
+export interface GraphCoordinates {
+  nodes: NodeCoordinates;
+  edges: Edges;
+}
+
+export interface NodeCoordinates {
+  [nodeName: string]: Node;
+}
+
+interface ExtractCoordinatesInterface {
```

Comment:
I don't think this interface is being used anywhere, do we want to remove it?

Reply:
Good catch! Will delete.